plugin/store/store.go

```diff
@@ -113,7 +113,7 @@ func (ps *PluginStore) getAllByCap(capability string) []CompatPlugin {
 
 	result := make([]CompatPlugin, 0, 1)
 	for _, p := range ps.plugins {
-		if _, err := p.FilterByCap(capability); err == nil {
+		if _, err := p.FilterByCap(capability); err == nil && p.PluginObj.Enabled {
```

Comment:
We want to return only enabled plugins from LookupWithCapability. This applies to FindWithCapability as well. So lets leave this function alone and move the check for enabled in LookupWithCapability and FindWithCapability.